### DISCUSSION TITLE
Can't bind default emacs keys without conflicts

### DISCUSSION THREAD

**Discussion by NicholasBHubbard** [meow-edit/meow#142] [Q&A] [answered]
Say for example I bind a command to `C-f`. Now when I hit the key bound to `meow-right` it runs the command bound to `C-f` instead of `meow-right`. This is because many of the meow commands are implemented to run as kbd-macros. 

I continue to run into these conflicts, and I don't understand why this would be desirable behavior. Is there any workaround for this or am I just using Meow wrong?

**Comment by DogLooksGood** [comment, marked as answer]
Hi, 
There's a variable called `meow--kbd-forward-char`, if you really have to change the keybinding, you have to change this variable.
For reason, I can give some examples:

1) if you want to forward a character, you use `forward-char`.  But if you are using vterm, the command on `C-f` is not `forward-char`, instead it is `vterm-send-C-f`. If we use `forward-char`, it will not work in vterm(But actually, currently adapting with vterm is not complete, forward word still using a command, not command on `M-f`.).

2) If you want to forward delete a char, you use `delete-char`.  But when using `smartparens`, it will be remapped to `sp-delete-char` which can keep the balance of parentheses.

Meow is designed to use with vanilla Emacs, especially with almost no tweaking on basic keybindings.
